abl-schema.json

```diff
@@ -92,7 +101,60 @@
       "required": [
         "repository_name",
         "style",
```

Comment:
Since the style will be stored inside the repo can this be moved inside the commit_metadata for git books?

[Card](https://github.com/openstax/ce/issues/1615) and [PR](https://github.com/openstax/richb-press/pull/15)

Reply:
Like so?
```json

        "commit_metadata": {
          "committed_at": "2021-10-25T10:47:06+00:00",
          "style": "english-composition",
          "books": [
            {
              "uuid": "ee7ce46b-0972-4b2c-bc6e-8998c785cd57",
              "slug": "writing-guide"
            }
          ]
        }

```